### ISSUE TITLE
libertine - no browser apps working

### ISSUE THREAD

**Issue by stekalb** [ubports/ubuntu-touch#791]
- Device: frieza
- Channel: dev
- Build: 20180723

### Steps to reproduce
Install any browser in libertine container:

**Comment by arubislander** [comment]
I can confirm that this is the case on:

Device: frieza
Channel: dev
Build: 20180725

**Comment by TartanSpartan** [comment]
Likewise with the newest RC image on arale for Firefox. This problem has been discussed on the forum recently, it is to do with broken Firefox builds for ARMHF architecture and the bug has been filed so hopefully a good fix comes in soon:

https://forums.ubports.com/topic/1430/firefox-crashing-on-startup-in-16-04-rc-dev

Chromium has never worked for me on any Vivid or on Xenial UT devices. Can anyone confirm if it works for any ARMHF architecture? I've never tried to install it on my Raspberry Pi.

I've never tried Epiphany, but will soon to test on both Vivid and Xenial.

**Comment by arubislander** [comment]
Indeed, I tried installing a version of firefox before 55. The only one currently in the xenial repo's is version 45. This one can be installed by giving ```firefox=45.0.2+build1-0ubuntu1``` as the package name.
This one does start OK, only thing is that it nags me that the browser if out of date (which it is).
